### ISSUE TITLE
Status of http://syriaca.org/person/635 Pseudo-Methodius?

### ISSUE THREAD

**Issue by davidamichelson** [srophe/srophe-app-data#398]
@nathangibson I am confused by record http://syriaca.org/person/635 which seems to be only the BHSE based half of a merge, but based our our URI should have had an author record too? Did we quarentine it somewhere due to needing to know how to handle pseudo-authors?

**Comment by davidamichelson** [comment]
Also, I see that the apocalypse is in the BHSE, but @jsaintlauren should we not include them? They are not really hagiography:                     
```
<idno type="BHS">1473</idno>
<idno type="BHS">1716</idno>
```

**Comment by davidamichelson** [comment]
Just to be clear, we would keep them in NHSL but not in BHSE

**Comment by jsaintlauren** [comment]
No they are not needed.

**Comment by nathangibson** [comment]
Yes, this is confusing. Here's what happened, as best I can tell: There are pseudo and anonymous authors mentioned in GEDSH/Barsoum/Abdisho, for whom we reserved URIs in a separate tab in the [Authors spreadsheet](https://docs.google.com/spreadsheets/d/1HPlPQDqp3lVlnSXQfNEy-9Q1IxtmbuZkro4ZNfDgwiQ/edit#gid=19) but did not make records as we were not ready to deal with them yet. It looks like I used these URIs to create records for three of them that BHSE mentions -- see  commit https://github.com/srophe/srophe-app-data/commit/65b90c7b02360539bc5f8ee60810444e4d7a8ae4 -- Clement of Rome (http://syriaca.org/person/423), Pseudo-Dionysius (http://syriaca.org/person/452), and Pseudo-Methodius (http://syriaca.org/person/635). 

**Some background before I suggest how to resolve:**
There was a point at which we thought we would make three person records in situations like this, all linked together: 1) the historical/hagiographical Methodius, 2) author of X, and 3) a Pseudo-Methodius placeholder. (See [Anonymous Authors Policy](https://docs.google.com/document/d/1NhwuazMJqOg8WVYyY8FYtBU2DzmLy2isGQhBlL00fXA/edit#) and [related diagram](https://docs.google.com/drawings/d/1HCmxcqilLmkew0e4dMynTLrZYzpqgBW_hh8zoCkf7hI/edit).) However, we later decided that the way to link them together is through the work (see comments in both of these docs). Thus, the work record should have an `author/@role='pseudo'` linking to the real Methodius and another author element linking to the "author of X" record. We still want a page where people can go to see "Pseudo-Methodius", but this would just be an artificially created viewport for works where Methodius is linked as "pseudo" author. 

**Suggested Resolution**
 1. Keep the original URIs reserved in the spreadsheet for the "author of X" type records. I'm not sure we need to create these records yet -- we could delete the ones I created such as person/635. Alternatively, we could just change them to the "author of X" format.
 2. Link the works (e.g., http://wwwb.library.vanderbilt.edu/exist/apps/srophe/work/1779) to the real person (e.g., http://wwwb.library.vanderbilt.edu/person/2800) using a "pseudo" `@role`, as I had already done for some.

**Comment by nathangibson** [comment]
Let me know what you think and what part of this you want me to do.

**Comment by dlschwartz** [comment]
@davidamichelson Should person/635 have `@ana="#syriaca-author"` alongside other `@ana` values?
